readthedocs-theme/templates/readthedocs/pricing.html

```diff
@@ -0,0 +1,504 @@
+                    <div class="header">Pageview analytics</div>
+                    <div class="description">90 days</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Auth auditing</div>
+                    <div class="description">90 days</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Single sign-on (SSO)</div>
+                    <div class="description">with Google</div>
+                  </div>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        </div>
+        <!-- end card -->
+
+      </div>
+
+      <div class="ui segment basic vertical padded">
+        <div class="ui horizontal divider header massive">
+          Base Plans
+        </div>
+      </div>
+      
+      <div class="ui three stackable cards">
+        
+        <!-- Card -->
+        <div class="ui card">
+
+          <div class="content">
+            
+            <div class="ui label ribbon blue big">Base</div>
+
+            <div class="ui header huge">Open Source</div>
+            <div class="meta">
+              <span class="category">forever free</span>
+            </div>
+
+            <button class="ui button fluid large blue">Signup</button>
+
+            <div class="description">
+              
+              <div class="ui list relaxed divided">
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Automatic build</div>
+                    <div class="description">on every commit</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Multi-version handling</div>
+                    <div class="description">from branches or tags</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Downloadable formats</div>
+                    <div class="description">PDF, HTML</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Enhanced full-text search</div>
+                  </div>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        </div>
+        <!-- end card -->
+        
+        <!-- Card -->
+        <div class="ui card">
+
+          <div class="content">
+            
+            <div class="ui label ribbon orange big">Ad-free</div>
+
+            <div class="ui header huge"><small><i>from</i></small> $5</div>
+            <div class="meta">
+              <span class="category">one-time donation</span>
```

Comment:
This one is an odd plan and a bit confusing. We should make an issue and fix this after. I think we're trying to sell reader users the ability not to see ads, so all of the RTD features are actually not useful in this plan.

Reply:
I opened an issue here:
* https://github.com/readthedocs/site-community/issues/40

For now, we could maybe go with a similar format to the "All open source plan features, plus:" only instead it would say "Upgrade to an Ad-free user, for a limited period of time, with all the features from the open source plan".

Reply:
Yeah, that actually sounds like a good compromise :+1:

Reply:
Great! Its done then. We can come back to it later through the issue above, when we have a better strategy to present it.